src/napari_spatialdata/_interactive.py

```diff
@@ -60,11 +89,27 @@ def __init__(self, sdata: SpatialData, headless: bool = False) -> None:
             self._sdata_widget, name="SpatialData", area="left", menu=self._viewer.window.window_menu
         )
         self._viewer.window.add_plugin_dock_widget("napari-spatialdata", "View")
+
         if not headless:
             self.run()
 
+        if tested_notebook is not None:
+            filepath = self.create_folders(tested_notebook, test_target)
+            fig, axs = plt.subplots(ncols=3, figsize=(12, 3))
+
+            for element_type, element_name, _ in self._sdata.filter_by_coordinate_system("global")._gen_elements():
```

Comment:
As suggested by @melonora in the other PR, the coordinate system could be passed to interactive. Please add an optional parameter, default to None, for doing this.